lib/modules/dosomething/dosomething_mbp/dosomething_mbp.module

```diff
@@ -148,10 +145,12 @@ function dosomething_mbp_get_transactional_payload($origin, $params = NULL) {
       $payload['merge_vars']['IMPACT_NUMBER'] = $params['impact_number'];
       $payload['merge_vars']['IMPACT_NOUN'] = $params['impact_noun'];
       $payload['merge_vars']['REPORTBACK_IMAGE_MARKUP'] = $params['image_markup'];
-      $payload['email_template'] = 'mb-campaign-report-back';
       $payload['email_tags'][] = 'drupal_campaign_reportback';
       break;
 
+    default:
+      watchdog('dosomething_mbp', 'Invslid dosomething_mbp_get_transactional_payload() $origin value: ' . $origin, WATCHDOG_ERROR);
```

Comment:
Wrong usage of `watchdog()`, see https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/watchdog/7